cloudfront/README.md

```diff
@@ -0,0 +1,21 @@
+CloudFront with dependencies such as WAF and ALB
```

Comment:
Please rename this module so it's clear what it does. It's a specific solution (CloudFront to ALB) so make up a clear name to give this module